README.md

```diff
@@ -39,7 +39,7 @@ Gradio is useful for:
 
 ## Quickstart
 
-**Prerequisite**: Gradio requires Python 3.7 or higher, that's all!
+**Prerequisite**: Gradio requires Python 3.8 or higher, that's all!
```

Comment:
There's a few other places where we mention 3.7:
- README badge (both on the main README as well as on the Chinese version)
- Contributing.md

Can we knock those out too?

Reply:
We also mention in `01_quickstart.md`

Reply:
Good catch - pushed this up now!